docs/config.md

```diff
@@ -25,7 +25,8 @@ connections:                                    # The list of connection public
 contracts: []                                   # The list of contract public ids the AEA project depends on (each public id must satisfy PUBLIC_ID_REGEX).
 protocols:                                      # The list of protocol public ids the AEA project depends on (each public id must satisfy PUBLIC_ID_REGEX).
 - fetchai/default:0.3.0
-skills:                                         # The list of skill public ids the AEA project depends on (each public id must satisfy PUBLIC_ID_REGEX).
+skills: []                                      # The list of skill public ids the AEA project depends on (each public id must satisfy PUBLIC_ID_REGEX).
+is_abstract: false                              # A boolean that if `true` makes the skill abstract, i.e. not instantiated by the framework but importable from other skills. Defaults to `false`. 
```

Comment:
This should be declared as an optional field here

Reply:
'Defaults to `false`" already suggests it is optional. I will make more clear.